vector/src/main/java/im/vector/adapters/VectorMessagesAdapterMediasHelper.java

```diff
@@ -374,6 +383,7 @@ public void onDownloadComplete(String aDownloadId) {
             downloadProgressLayout.setVisibility(View.GONE);
         }
 
+        imageView.setLayerType(View.LAYER_TYPE_SOFTWARE, null);
```

Comment:
Is it required by ion or is it old code to make the webview work ;) ?

If it is required, it would be better to do it only for stickers to avoid performance decrease.